static/usage.html

```diff
@@ -350,7 +350,7 @@ <h2><a href="#contact-scopes">Contact Scopes</a></h2>
                 <h2><a href="#accessibility">Accessibility</a></h2>
 
                 <p>GrapheneOS includes all of the accessibility features from the Android Open
-                Source Project and strives to fill in the gaps from not including Google apps and
+                Source Project and strives to fill in the gaps by not including Google apps and
```

Comment:
Based on my understanding, the original wording is correct. There are gaps from not including proprietary Google apps. Gaps aren't being filled by excluding Google apps.

Reply:
I would say the opposite, but tbf I trust you since english is not my main language.

Reply:
Yes, I think keep the original unchanged.